target/linux/generic/pending-6.6/779-net-vxlan-don-t-learn-non-unicast-L2-destinations.patch

```diff
@@ -17,9 +17,9 @@ Signed-off-by: David Bauer <mail@david-bauer.net>
 
 --- a/drivers/net/vxlan/vxlan_core.c
 +++ b/drivers/net/vxlan/vxlan_core.c
-@@ -1446,6 +1446,10 @@ static bool vxlan_snoop(struct net_devic
- 	struct vxlan_fdb *f;
- 	u32 ifindex = 0;
+@@ -1450,6 +1450,10 @@ static bool vxlan_snoop(struct net_devic
+ 	if (!is_valid_ether_addr(src_mac))
+ 		return true;
  
 +	/* Don't learn broadcast packets */
 +	if (is_multicast_ether_addr(src_mac) || is_zero_ether_addr(src_mac))
```

Comment:
This patch can be removed. It will never return false. And the uptream commit should fix the same issue: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/commit?h=v6.6.35&id=924f7bbfc5cfd029e417c56357ca01eae681fba6

Reply:
@graysky2 @DragonBluep It looks like the upstream has been fixed and the patch can be removed.

Reply:
done